RussianFilter/sections/antiadblock.txt

```diff
@@ -20,7 +20,7 @@ electric-house.ru#@#.adsbygoogle
 !
 !
 ! https://github.com/AdguardTeam/AdguardFilters/issues/100466
-cdnpotok.com#?##oframeplayerjs > pjsdiv[style]:has(> pjsdiv[style] > p[style])
+cdnpotok.com#?##oframeplayerjs > pjsdiv[style]:has(> pjsdiv[style] > p[style]:contains(/AdBlock/))
```

Comment:
Do you need regex here?

Reply:
Yes, on my opinion. Because the whole phrase is very long.

<details><summary>Screenshot:</summary>

<img width="1214" alt="Снимок экрана 2021-12-03 в 17 10 19" src="https://user-images.githubusercontent.com/91964807/144616230-1ebd4feb-2f94-414b-af61-02521f69312b.png">

</details><br/>

Reply:
According to your screenshot, you can just use `:contains(AdBlock)`.
New rule
```
cdnpotok.com#?##oframeplayerjs > pjsdiv[style]:has(> pjsdiv + pjsdiv[style] > p[style]:contains(AdBlock))
```